CAIPs/caip-25.md

```diff
@@ -137,9 +137,11 @@ The valid error messages codes are the following:
     * code = 5102
     * message = "Requested events are not supported"
 
-## Links
+## Changelog
 
-n/a
+- 2022-11-26: add mandatory indexing by session identifier (i.e. CAIP-171 requirement) 
```

Comment:
It's a Draft, imho the changelog shouldn't be in it until it's Final, where we could add Errata.